host/4/publish-appservice-stage.yml

```diff
@@ -51,6 +51,18 @@ steps:
     displayName: tag and push dotnet images
     continueOnError: false
 
+  - bash: |
+      set -e
+      docker pull $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-dotnet8
+
+      docker tag $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-dotnet8 $TARGET_REGISTRY/dotnet:$(TargetVersion)-dotnet8-stage$(StageNumber)
```

Comment:
This pipeline is intended to release appservice images. I think we need another definition that includes appservice stuff I will  leave another comment

Reply:
added